content/bd/0004-relational-model-conversion.md

```diff
@@ -376,28 +371,22 @@ recorrer a **Restrições de Integridade**.
 
 ![Diagrama do Modelo E-A de uma associação many-to-many com participação obrigatória](./assets/0004-association-many-to-many-mandatory.svg#dark=3)
 
-(atributos das entidades omitidos por brevidade)
-
 > teacher(<u>ist_id</u>, name)
 >
 > course(<u>course_id</u>, course_name, department)
 >
-> - IC-1: Every course (course_id) must participate in the lectures association
->
-> lectures(<u>ist_id, course_id</u>)
->
+> - IC-1: Every course (_course_id_) must participate in the lectures association
+
```

Comment:
```suggestion
> - IC-1: Every course (_course_id_) must participate in the lectures association
>
> lectures(<u>ist_id</u>, <u>course_id</u>)
>
```
What was the point of deleting this relation? I don't get it...

Reply:
Oh, I guess that was inadvertently erased. My bad.